### DISCUSSION TITLE
Add volume block to text to voice extension

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7824] [Ideas]
The text to voice extension is fantastic, but whereas the sound category has a perfectly working volume block, the text to voice extension doesn't allow to fine tune the volume. So all voices have the same volume, unless you operate the ScratchAddons' volume  control or the PC's.

**Comment by mxmou** [comment]
This unfortunately isn't possible - the block wouldn't work if the project was viewed by users who don't have Scratch Addons.

Related discussion in a Scratch repo: https://github.com/scratchfoundation/scratch-vm/issues/2526
